src/HttpServer.ts

```diff
@@ -254,6 +255,68 @@ export class HttpServer {
         }
     }
 
+    /**
+     * @openapi
+     * /estimate:
+     *   post:
+     *     summary: It estimate the mas possible gas in relay transaction.
+     *     description: It receives transactions to be estimated (deploy or forward requests) and after performing all the checks, it estimates the gas consumption.
+     *     requestBody:
+     *       description: Deploy transaction or forward transaction.
+     *       required: true
+     *       content:
+     *         application/json:
+     *           schema:
+     *             oneOf:
+     *               - $ref: '#/components/schemas/DeployTransactionRequest'
+     *               - $ref: '#/components/schemas/RelayTransactionRequest'
+     *           examples:
+     *             deploy:
+     *               summary: "Deploy request example"
+     *               value: {"relayRequest":{"request":{"relayHub":"0x66Fa9FEAfB8Db66Fe2160ca7aEAc7FC24e254387","to":"0x0000000000000000000000000000000000000000","data":"0x","from":"0xCB8F8130E289946aec9a17b29819716B1E9e4998","value":"0","nonce":"5","tokenAmount":"0","tokenGas":"0x00","tokenContract":"0xF5859303f76596dD558B438b18d0Ce0e1660F3ea","recoverer":"0x0000000000000000000000000000000000000000","index":"6"},"relayData":{"gasPrice":"65164000","callVerifier":"0x5C6e96a84271AC19974C3e99d6c4bE4318BfE483","domainSeparator":"0xa81483953da7601ef828906dbab2e4baf21ddfd3d3c484fe7c43c55836c6c772","callForwarder":"0xeaB5b9fA91aeFFaA9c33F9b33d12AB7088fa7f6f","relayWorker":"0x74105590d404df3f384a099c2e55135281ca6b40"}},"metadata":{"relayHubAddress":"0x66Fa9FEAfB8Db66Fe2160ca7aEAc7FC24e254387","signature":"0x1285a1fa3217b0b8ca9b23ee2404324c965af9bb3908199ffc8bc7a47f59cef9160a142da5269fa5b7bfa8a688c1a507bedeba0650f1d617b93c8ece598aba651c","relayMaxNonce":30}}
```

Comment:
I think we should replace the `relayWorker` with `feesReceiver`. Likely we have the same issue with the other requests also (not fixed before)

Reply:
updated